src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -63,6 +63,12 @@ std::pair<PlasmaObject, PlasmaError> CreateRequestQueue::TryRequestImmediately(
     const CreateObjectCallback &create_callback, size_t object_size) {
   PlasmaObject result = {};
 
+  // Immediately fulfill it using the fallback allocator.
```

Comment:
Hmm actually I'm confused about how this fits with #16523. I thought we would just remove the TryRequestImmediately path?

Reply:
It's still used for allocating error messages, which I think shouldn't ever wait on spilling.